test-apps/appui-test-app/connected/src/frontend/index.tsx

```diff
@@ -2,6 +2,7 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+import "@itwin/itwinui-react/styles.css";
```

Comment:
The FallbackStyles loader implemented by iTwinUI is not great, because the styles are loaded after rendering the page, hence some flickering is visible. Our storybook is set up to use that mechanism.